system-int-test/test/cmr/system_int_test/search/generics_test.clj

```diff
@@ -38,28 +30,28 @@
    (-> {:method :get
         :url (format "%s%s?%s" (url-helper/search-root) concept-type-ext params)
         :connection-manager (system/conn-mgr)
-        :throw-exceptions false
-        :headers { transmit-config/token-header
-                  (transmit-config/echo-system-token)}}
-       (clj-http.client/request))))
+        :throw-exceptions false}
+       (client/request))))
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; Tests
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+;; Test that a Generic can be searched and have the search results use XML.
 (deftest test-search-results
-  "Test that a Generic can be searched and have the search results use XML."
 
   (let [native-id (format "Generic-Test-%s" (UUID/randomUUID))
-        ;generic-requester (partial generic-request "grid" "PROV1" native-id)
-        generic-requester (partial gen-util/generic-request
-                                   (transmit-config/echo-system-token)
-                                   "PROV1"
-                                   native-id
-                                   "grid")
+        generic-requester (partial gen-util/generic-request "PROV1" native-id "grid")
         good-generic-requester (partial generic-requester gen-util/grid-good)
-        post-results (good-generic-requester :post)]
+        post-results (good-generic-requester :post)
+        body (json/parse-string (:body post-results) true)
+        _ (println body)
```

Comment:
We should remove the debugs from the final test.

Reply:
Doh!  I removed them.